source/signup_form.php

```diff
@@ -47,10 +48,10 @@
         </p>
         <p>
             <input type="hidden" name="csrf_token" value="<?php echo h(setToken()); ?>">
-            <input type="submit" value="新規登録">
+            <input type="submit" value="新規登録" class="btn btn-primary">
```

Comment:
テスト用のコードが存在しているので削除して下さい。
```suggestion
            <input type="submit" value="新規登録">
```